### PR TITLE
Add Form Builder production namespace

### PR DESCRIPTION
**Pull request by solidgoldpig**

### CODE REVIEW

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-production/resources/publisher.tf [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    url = "postgres://${module.publisher-rds-instance.database_username}:${module.publisher-rds-instance.database_password}@${module.publisher-rds-instance.rds_instance_endpoint}/${module.publisher-rds-instance.database_name}"
+  }
+}
+
+##################################################
+
+########################################################
+# Publisher Elasticache Redis (for resque + job logging)
+module "publisher-elasticache" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster?ref=2.0"
```
Comment (alkar) [inline namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-production/resources/publisher.tf]: Can you please update to version `2.1` for the elasticache cluster module?

Comment (solidgoldpig) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-production/resources/publisher.tf]: Done

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-services-staging/01-rbac.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: formbuilder-services-staging-admins
-  namespace: formbuilder-services-staging
+  name: formbuilder-services-dev-admins
```
[the remark this answers is not in the document]
Comment (solidgoldpig) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-services-staging/01-rbac.yaml]: No. Thanks for the catch

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-production/01-rbac.yaml [hunk 3]
```diff
@@ -1,12 +1,11 @@
 
-
 # Bind admin role for namespace to team group
 ---
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: formbuilder-platform-staging-admins
-  namespace: formbuilder-platform-staging
+  name: formbuilder-platform-production-admins
```
[the remark this answers is not in the document]
Comment (solidgoldpig) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-production/01-rbac.yaml]: Can't explain Git's take on the provenance, but there's a 01-rbac.yml in both the staging and production directories and the values look ok to me

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-services-production/01-rbac.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: formbuilder-services-production-service-account
+  namespace: formbuilder-services-production
+subjects:
+  # dev, staging and production Publisher service accounts
+  # are admins, so that they can both deploy to either
+  - kind: ServiceAccount
+    name: formbuilder-publisher-workers-dev
+    namespace: formbuilder-platform-dev
```
[the remark this answers is not in the document]
Comment (solidgoldpig) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-services-production/01-rbac.yaml]: Yes.

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-services-dev/01-rbac.yaml [hunk 5]
```diff
@@ -20,14 +20,17 @@ metadata:
...
-  # both dev & staging Publisher service accounts
+  # dev, staging and production Publisher service accounts
   # are admins, so that they can both deploy to either
   - kind: ServiceAccount
     name: formbuilder-publisher-workers-dev
     namespace: formbuilder-platform-dev
   - kind: ServiceAccount
     name: formbuilder-publisher-workers-staging
     namespace: formbuilder-platform-staging
+  - kind: ServiceAccount
```
[the remark this answers is not in the document]
Comment (solidgoldpig) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-services-dev/01-rbac.yaml]: Yes.

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-production/01-rbac.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (alkar) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-production/01-rbac.yaml]: The file is definitely gone from `formbuilder-platform-staging/formbuilder-platform-staging-admin-role.yaml`. Perhaps you haven't committed it?

**Comment by solidgoldpig** [comment]
The elasticcache refs are all updated now. `formbuilder-platform-staging/formbuilder-platform-staging-admin-role.yaml`  was duplicating the roles declared on `01-rbac.yaml`

**Pull request merged**